IDE/Espressif/ESP-IDF/examples/wolfssl_benchmark/main/main.c

```diff
@@ -132,12 +134,13 @@ char* __argv[WOLFSSL_BENCH_ARGV_MAX_ARGUMENTS];
 
 int construct_argv()
 {
+    #define ARG_BUFF_SIZE 16
```

Comment:
Move #define out of function.
Putting just above is fine.

Reply:
moved.